.github/workflows/testing.yml

```diff
@@ -31,13 +88,13 @@ jobs:
         if: matrix.buildsys == 'make'
         run: |
           export PATH="/usr/local/opt/bison/bin:$PATH"
-          make develop -j Q= CXX=${{ matrix.cxx }}
+          make develop -ej CXXFLAGS="-ggdb3 -Og -fno-omit-frame-pointer -fno-optimize-sibling-calls -mmacosx-version-min=10.9 -I include/hacks" Q=
```

Comment:
You can see this does not work:

```bash
c++ -ggdb3 -Og -fno-omit-frame-pointer -fno-optimize-sibling-calls -Wall -Wno-unknown-warning-option -Wno-c99-designator -Werror -Wextra 	-Walloc-zero -Wcast-align -Wcast-qual -Wduplicated-branches -Wduplicated-cond 	-Wfloat-equal -Wlogical-op -Wnull-dereference -Wshift-overflow=2 	-Wstringop-overflow=4 -Wstrict-overflow=5 -Wundef -Wuninitialized -Wunused 	-Wshadow 	-Wformat=2 -Wformat-overflow=2 -Wformat-truncation=1 	-Wno-format-nonliteral -Wno-strict-overflow 	-Wno-type-limits -Wno-tautological-constant-out-of-range-compare 	-Wvla 	-D_GLIBCXX_ASSERTIONS 	-fsanitize=shift -fsanitize=integer-divide-by-zero 	-fsanitize=unreachable -fsanitize=vla-bound 	-fsanitize=signed-integer-overflow -fsanitize=bounds 	-fsanitize=object-size -fsanitize=bool -fsanitize=enum 	-fsanitize=alignment -fsanitize=null -fsanitize=address -x c++ -std=c++2a -I include -D_POSIX_C_SOURCE=200809L -fno-exceptions -fno-rtti -c -o src/extern/getopt.o src/extern/getopt.cpp
```

... because `make develop` overrides `CXXFLAGS`. `CXXFLAGS` must be passed as an environment variable so that it overrides `make develop`'s own override (because of `-e`).

I hope you like millefeuilles. (I don't, and I don't like Make either. :')

Reply:
I did not look closely enough; CI passed when I passed it to `make` directly. A comment noting why it's done this way to get around `make develop` would be helpful.

Reply:
Unfortunately `make develop` doesn't pass `-e` either! I thought it would, but it doesn't. I'm not sure how to get this working...

Reply:
Problem obsoleted by [the new strategy](https://github.com/gbdev/rgbds/pull/1280#discussion_r1437941893).